package.json

```diff
@@ -21,7 +21,8 @@
     "jest": "jest --config config/jest/test.config.js",
     "lint:watch": "esw --watch --config ./eslint/eslintrc packages",
     "lint": "eslint --config .eslintrc.js packages",
-    "publish": "node utils/pre-publish-check.js && git diff --stat --exit-code HEAD && yarn build:cjs && yarn build:es6 && lerna publish",
+    "publish": "node utils/pre-publish-check.js && git diff --stat --exit-code HEAD && yarn build:all && lerna publish",
+    "release": "node utils/pre-publish-check.js && git diff --stat --exit-code HEAD && yarn build:all && changeset publish",
```

Comment:
Can we rename these to `publish:lerna` and `publish:changesets`?  Someone looking at this might think that these two commands do different things, but really they both publish, but use different tools to do so.

Reply:
I'm going to use `publish:ci` so we follow the `*:ci` pattern that @somewhatabstract recently introduced in the repo :).